docs/CONTRIBUTING.md

```diff
@@ -0,0 +1,256 @@
+Unfortunately, this isn't visible to the public because GitHub requires that you
+be a member of the Organization to see Organization-level Projects. If you want
+to get involved in this process, get in touch with us on Slack!
+
+We currently don't have this process formalized, so feel free to contribute to
+this document itself!
+
+[Nerves Radar]: https://github.com/orgs/nerves-project/projects/1
+
+## Code and Documentation changes
+
+If you're looking for a way to directly contribute to the code and
+documentation, check our [repositories] for Issues that are labeled with
+`kind:documentation`, `kind:bug`, or `kind:enhancement`. You can use the
+`level:stater`, `level:intermediate`, and `level:advanced` labels to determine
+how difficult we think it will be, whether you're looking for an easy first-time
+contribution or an extra challenge.
+
+If you're working on an Issue, feel free to set yourself as the **Assignee**,
+and don't feel bad about removing yourself if you later decide that you're not
+going to work on it. This helps us keep track of which Issues are currently
+being worked-on, avoiding duplicated effort and Issues that get stale over time.
+
+Working on your first Pull Request? You can learn how from this *free* series,
+[How to Contribute to an Open Source Project on GitHub].
+
+[How to Contribute to an Open Source Project on GitHub]: https://egghead.io/series/how-to-contribute-to-an-open-source-project-on-github
+
+### Copyright assignment
+
+**TODO:
+  I mostly made this up based on http://requirejs.org/docs/contributing.html.
+  Review CLA/DCO processes for other OSS projects**
+
+Licensing is very important to open-source software projects, especially for
+companies who want to use that software in a commercial product. By contributing
+to the Nerves source code or documentation, you are agreeing to release your own
+intellectual property (and not the intellectual property of others) to the
+Nerves project.
+
+### Code review process
+
+The core team has GitHub integrations with Slack, so we will be notified immediately in the #nerves-dev channel about activity on GitHub Issues and Pull Requests. Our goal is to at least reply with a comment within a few days of a new Issue or Pull Request being opened to let you know what we saw it and we're thinking about it.
+
+We currently don't have a formal code-review process in place, but our current
+guidelines are as follows:
+
+* Each repository should have an owner and one or more maintainers.
+* At least one approval is required to merge a Pull Request.
+* Anyone on the core team can approve a Pull Request on any repository.
+
+  > NOTE: Please approve responsibly. If you're not deeply familiar with the
+  code, feel free to comment or request changes, but don't approve.
+
+* The owner of a repository should be the one to perform the merge.
+
+  > NOTE: We prefer the GitHub "Squash and Merge" button, but any method is OK.
```

Comment:
I actually have a strong opinion against "Squash and Merge". I prefer "Rebase and Merge" so that the commit history remains linear and each commit is one logical change. Anyway, doesn't matter, since it would be good to move this section anyway.

Reply:
This is an example of why we should have these guidelines posted somewhere. ;)